solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1278,6 +1287,15 @@ public Builder withMaxConnectionsPerHost(int max) {
       return this;
     }
 
+    /**
```

Comment:
This should have both a "@param" and a "@return" line as part of the javadoc, with a few words describing each.